app/src/main/java/br/org/cesar/knot_setup_app/fragment/gatewayConnected/GatewayConnectedFragment.java

```diff
@@ -0,0 +1,112 @@
+package br.org.cesar.knot_setup_app.fragment.gatewayConnected;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.nsd.NsdManager;
+import android.net.nsd.NsdServiceInfo;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.support.v4.app.Fragment;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ListView;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
+import br.org.cesar.knot_setup_app.R;
+import br.org.cesar.knot_setup_app.activity.splash.SplashActivity;
+import br.org.cesar.knot_setup_app.domain.adapter.GatewayAdapter;
+import br.org.cesar.knot_setup_app.fragment.gatewayConnected.GatewayConnectedContract.ViewModel;
+import br.org.cesar.knot_setup_app.fragment.gatewayConnected.GatewayConnectedContract.Presenter;
+import br.org.cesar.knot_setup_app.utils.Constants;
+import br.org.cesar.knot_setup_app.wrapper.LogWrapper;
+
+public class GatewayConnectedFragment extends Fragment implements ViewModel {
+
+    private View mView;
+    private ListView deviceListView;
+
+    private Presenter mPresenter;
+    private ArrayList<NsdServiceInfo> deviceList;
+    private GatewayAdapter adapter;
+    private Handler mHandler;
+
+
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_list, container,false);
+
+        mView = view;
+        mPresenter = new GatewayConnectedPresenter(this,
+                (NsdManager) getActivity().getSystemService(Context.NSD_SERVICE),getContext());
+        deviceList = new ArrayList<>();
+        adapter = new GatewayAdapter(getActivity(), R.layout.item_gateway, deviceList);
+        mHandler = new Handler(Looper.getMainLooper());
+
+        deviceListView = view.findViewById(R.id.list);
+        setAdapter();
+
+        return view;
+    }
+
+    /**
```

Comment:
If you're overriding a method, the javadoc should be in the initial declaration.